client/packages/common/src/plugins/types.ts

```diff
@@ -1,11 +1,13 @@
 export enum PluginArea {
   AppBar,
+  Dashboard,
   Toolbar,
 }
 
 export enum PluginType {
   InboundShipment,
   InternalOrder,
+  None,
```

Comment:
Added this for the plugins which don't require data to be passed in

Reply:
This is not clear to me, do you have an example?

Reply:
The [dashboard plugins](https://github.com/openmsupply/open-msupply/tree/feature/front-end-plugins-example-01) are making use of this - I had assumed that all plugins would require context-related data.. but the dashboard is an exception. The components are fetching stats independently

Reply:
Maybe you could call it Dashboard? and in PluginArea you call it DashboardWidget or similar?

Reply:
I do like that - but it does mean that `Dashboard` implies no data. I think it is better to explicitly state `No data` in some way. `Fetches it's own data`? 
Oh wait.. perhaps the issue is that I've called it `PluginType` when I had in my mind `PluginDataType`. I'll change it to `Dashboard` 🙏